src/components/Paypal.js

```diff
@@ -58,6 +70,19 @@ const Paypal = ({ notify }) => {
                 >
                   {__("Manage", "wp-module-ecommerce")}
                 </Button>
+              ) : typeof yith_ppwc_login !== "undefined" &&
+                !parseInt(yith_ppwc_login?.liveConnected) ? (
+                <Link
+                  href={yith_ppwc_login.loginURL}
+                  variant="primary"
+                  style={{ color: "#fff" }}
```

Comment:
@sumathi3399 Is it necessary to specify `color` inline? is it not included in `primary` styling

Reply:
removed the inline styling and added the text color in className